pkgs/build-support/node/build-npm-package/hooks/npm-build-hook.sh

```diff
@@ -0,0 +1,15 @@
+npmBuildHook() {
+    echo "Executing npmBuildHook"
+
+    runHook preBuild
+
+    echo "Running build script"
```

Comment:
Doesn't `npm run` output something when its starting to build? If yes, this log seems redundant to me. If not i think this is a bit more descriptive:
```suggestion
    echo "Running npm build script"
```